composer.json

```diff
@@ -12,18 +12,18 @@
     "require": {
         "php": "^5.3.9|^7.0",
         "webmozart/path-util": "^2.2",
-        "webmozart/glob": "^4.0",
+        "webmozart/glob": "^4.1",
         "webmozart/assert": "^1.0"
     },
     "require-dev": {
-        "webmozart/key-value-store": "^1.0.0-beta5",
```

Comment:
You can add it back in in the respective PR, both in "require-dev" and "suggest" (like in puli/discovery).